pkgs/tools/typesetting/tex/dblatex/default.nix

```
@@ -21,14 +21,15 @@ assert enableAllFeatures ->
   ghostscript != null;
 
 stdenv.mkDerivation rec {
-  name = "dblatex-0.3.11";
+  pname = "dblatex";
+  version = "0.3.12";
 
   src = fetchurl {
-    url = "mirror://sourceforge/dblatex/${name}.tar.bz2";
-    sha256 = "0rp1bc2lgisigscq1i7zxfd2qdaxxxld6khbcxss4pq7fpi9fzkv";
+    url = "mirror://sourceforge/dblatex/${pname}3-${version}.tar.bz2";
+    sha256 = "0yd09nypswy3q4scri1dg7dr99d7gd6r2dwx0xm81l9f4y32gs0n";
```

```suggestion
    sha256 = "1vc97w0kvchxzlf0v7nn1fzx7pi0ih29cdqd6r7rg1y6parqa6i5";
```

According to ofborg but my first build succeeded. Not sure what happened here.

---

Different mirrors serving different contents, maybe? We can try fetching from their mercurial repository instead…

---

> Different mirrors serving different contents, maybe?

Probably or I get some html file because the mirror did not have the archive. Now it builds.